lib/vizion.js

```diff
@@ -45,8 +45,11 @@ vizion.revertTo = function(argv, cb) {
   var revision = (argv.revision) ? argv.revision : false;
   var _folder = (argv.folder != undefined) ? argv.folder : '.';
 
-  if (!revision)
+  if (!revision) {
     return cb({msg: 'Cannot revert to an invalid commit revision', path: _folder});
+  } else if (!/^[A-Fa-f0-9]+$/.test(revision)) {
+    return cb({msg: 'Cannot revert to an invalid commit revision', path: _folder});
+  }
```

Comment:
I don't understand why the condition return the same result.

Reply:
Yeah, I might've been drunk(jk) when I wrote that code, it could be rewritten as
```js
if (!revision || !/^[A-Fa-f0-9]+$/.test(revision)) return cb({msg: 'Cannot revert to an invalid commit revision', path: _folder});
```
Let me update it !